php/libraries/User.class.inc

```diff
@@ -495,6 +495,9 @@ class User extends UserPermissions
 
     /**
      * Check if user belongs to DCC
+     * FIXME 'CenterID' is no longer used.
+     *       This should check that the 'CenterIDs' array cointains a 1
```

Comment:
Why not implement this? 😄

Reply:
I just wanted to comment first to see if anyone had any reasons for me not to. Will do now!